### PR TITLE
Automated Test - COMMERCE-6085 | master

### PR DESCRIPTION
**Pull request by brittneyq**
https://issues.liferay.com/browse/COMMERCE-6085

cc: @pietro69

### CODE REVIEW

**Comment by brittneyq** [comment]
ci:test:sf

**Comment by brittneyq** [comment]
ci:test:commerce-es7-remote

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductSubscriptions.testcase [hunk 1]
```diff
@@ -38,13 +38,27 @@ definition {
...
+			category = "Product Management",
+			portlet = "Products");
 
+		CPCommerceCatalog.newProductsEntry(
+			productName = "Simple T-Shirt",
+			productType = "Simple");
+
+		CommerceEntry.gotoMenuTab(menuTab = "Subscription");
+
+		Click(locator1 = "CommerceEntry#PAYMENT_SUBSCRIPTION_TOGGLE");
```
Comment (Pietro69) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductSubscriptions.testcase:55]: For the locator i think is possible to use the macro Check.checkToggleSwitch with:
key_toggleSwitchLabel = "Payment Subscription"
locator1 = "ToggleSwitch#ANY_TOGGLE_SWITCH"

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductSubscriptions.testcase [hunk 2]
```diff
@@ -38,13 +38,27 @@ definition {
...
+
+		Click(locator1 = "CommerceEntry#DELIVERY_SUBSCRIPTION_TOGGLE");
```
Comment (Pietro69) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductSubscriptions.testcase:57]: For the locator i think is possible to use the macro Check.checkToggleSwitch with:
key_toggleSwitchLabel = "Delivery Subscription"
locator1 = "ToggleSwitch#ANY_TOGGLE_SWITCH"

File: modules/apps/commerce/commerce-product-test/src/testFunctional/paths/CommerceEntry.path [hunk 3]
```diff
@@ -509,6 +509,16 @@
...
+<tr>
+	<td>PAYMENT_SUBSCRIPTION_TOGGLE</td>
+	<td>//h4[contains(text(), 'Payment Subscription')]//following-sibling::span[contains(@class,'toggle-switch-handle')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>DELIVERY_SUBSCRIPTION_TOGGLE</td>
+	<td>//h4[contains(text(), 'Delivery Subscription')]//following-sibling::span[contains(@class,'toggle-switch-handle')]</td>
+	<td></td>
+</tr>
```
Comment (Pietro69) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/paths/CommerceEntry.path:521]: I think is possible to remove it

**Review by Pietro69 — CHANGES_REQUESTED** [review]

**Comment by Pietro69** [comment]
Please add the property test.name.skip.portal.instance = "CPCommerceProductSubscriptions#UseSubscriptions"

**Comment by brittneyq** [comment]
ci:test:sf

**Comment by brittneyq** [comment]
ci:test:sf

**Comment by brittneyq** [comment]
fixed @Pietro69

**Comment by Pietro69** [comment]
ci:test:commerce-functional

**Comment by Pietro69** [comment]
ci:forward

**Pull request closed without merging**